Base/res/ladybird/inspector.js

```diff
@@ -204,7 +204,16 @@ inspector.createPropertyTables = (computedStyle, resolvedStyle, customProperties
         newTable.setAttribute("id", tableID);
 
         Object.keys(properties)
-            .sort()
+            .sort((a, b) => {
+                let baseResult = a.localeCompare(b);
+                // Manually move vendor-prefixed items after non-prefixed ones.
+                if (a[0] === "-") {
+                    if (b[0] === "-") return baseResult;
```

Comment:
I know this is prettier's fault for one-lining this, but I find it pretty yuck. Can we add braces?
```js
if (b[0] === "-") {
    return baseResult;
}
```

Reply:
Done! Yeah I wasn't a fan of that either.